tests/playwright/server.js

```diff
@@ -18,6 +17,42 @@ app.get('/asset-paths', (req, res) => {
   res.sendFile(path.join(__dirname, 'fixtures/asset-paths.html'));
 });
 
+app.get('/asset-paths/query-params', (req, res) => {
+  res.sendFile(path.join(__dirname, 'fixtures/asset-paths/query-params.html'));
+});
+
+app.get('/asset-paths/asset-at-directory-name', (req, res) => {
+  res.sendFile(path.join(__dirname, 'fixtures/asset-paths/asset-at-directory-name.html'));
+});
+
+app.get('/asset-paths/relative-path', (req, res) => {
+  res.sendFile(path.join(__dirname, 'fixtures/asset-paths/relative-path.html'));
+});
+
+app.get('/asset-paths/long-file-name', (req, res) => {
+  res.sendFile(path.join(__dirname, 'fixtures/asset-paths/long-file-name.html'));
+});
+
+app.get('/asset-paths/external-asset-archived', (req, res) => {
+  res.sendFile(path.join(__dirname, 'fixtures/asset-paths/external-asset-archived.html'));
+});
+
+app.get('/asset-paths/external-asset-not-archived', (req, res) => {
+  res.sendFile(path.join(__dirname, 'fixtures/asset-paths/external-asset-not-archived.html'));
+});
+
+app.get('/asset-paths/css-urls', (req, res) => {
+  res.sendFile(path.join(__dirname, 'fixtures/asset-paths/css-urls.html'));
+});
+
+app.get('/asset-paths/percents', (req, res) => {
+  res.sendFile(path.join(__dirname, 'fixtures/asset-paths/percents.html'));
+});
+
+app.get('/asset-paths/srcset', (req, res) => {
+  res.sendFile(path.join(__dirname, 'fixtures/asset-paths/srcset.html'));
+});
```

Comment:
There's got to be a better way to do this. Is there some magic "serve all files in this directory" thing (since the requested path and served path are always the same for these)?

Reply:
You could do something like this:
```
app.get('/asset-paths/:page', (req, res) => {
  res.sendFile(path.join(__dirname, `fixtures/asset-paths/${req.params.page}`));
});
```